doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -24,7 +24,6 @@
 import no.unit.nva.model.contexttypes.Book;
```

Comment:
JUnit5 visibility for tests: package private

Reply:
a little cryptic...